app/app/urls.py

```diff
@@ -62,8 +62,8 @@
     # oauth2 provider
     url('^o/', include('oauth2_provider.urls', namespace='oauth2_provider')),
     path('jsi18n/', JavaScriptCatalog.as_view(), name='javascript-catalog'),
-    # create bounty
-    url(r'^create?', dashboard.views.new_bounty, name='create_bounty'),  # TODO: Remove
```

Comment:
This definitely isn't used anywhere? or by some weird 3rd party integration that we forgot about or something?

Reply:
Yeah ! This was introduced as a means to have multiple ways to access the same way !
the 3rd part were with -> `/new` OR `/fund` (cause one of those were added cause of that )

I don't know if that 3rd part integration even exists (separate story all together )